templates/artist/artist_list.html

```diff
@@ -10,6 +10,7 @@
     <div class="break">
         <hr>
     </div>
+    <div id="filter-options">
```

Comment:
There's already a `<div>` here for `.filter-sort-options`. You should be able to just reuse this class for the new stylings that you've applied to `#filter-options`.